cloudbuild.yaml

```diff
@@ -0,0 +1,49 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+steps:
+  - name: 'gcr.io/cloud-builders/docker'
+    id: 'Build local images for Airflow CI - Python 2.7'
+    entrypoint: '/bin/bash'
+    waitFor: ['-']
+    env:
+      - "CI=true"
+      - "PYTHON_VERSION=2.7"
+      - "DOCKERHUB_USER=potiuk"
```

Comment:
:)

Reply:
Yeah. I am testing it now using my own DockerHub account :). Will remove it eventually :)

Reply:
As discussed before - for now "potiuk" is my DockerHub user that I need for testing. Will switch back to airflow closer to when we would like to merge.